galley/pkg/config/analysis/diag/level.go

```diff
@@ -68,3 +69,36 @@ func GetUppercaseStringToLevelMap() map[string]Level {
 	}
 	return m
 }
+
+// MessageThreshold is a wrapper around Level to use as cobra command line arguments
+type MessageThreshold struct {
+	Level
+}
```

Comment:
1) I don't really understand why we need another struct for this. A Level _is_ a threshold, and there's no additional type safety gained here.

2) FYI: You can just ```type MessageThreshold Level```

Reply:
The MessageThreshold struct is one of the codes which I kept as was, but this struct is only used by istioctl analyze for now so yeah I could merge it with Level.